packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js

```diff
@@ -2061,10 +2079,10 @@ function describeNestedLists(childList: {
 
 const styles = StyleSheet.create({
   verticallyInverted: {
-    flexDirection: 'column-reverse',
+    transform: [{scaleY: -1}],
   },
   horizontallyInverted: {
-    flexDirection: 'row-reverse',
+    transform: [{scaleX: -1}],
```

Comment:
How come we're back to this? I know this was originally what we had

Reply:
Reverted this as `flexDirection` doesn't work well for sticking scroll view to the bottom. We are now applying a combinatioin of this and `contentContainerStyle` to achieve correct inverted scroll behaviour.

Reply:
specifically on webkit somehow